src/components/UserInfo/UserInfo.scss

```diff
@@ -0,0 +1,3 @@
+.UserInfo {
+  font-style: italic;
```

Comment:
Consider using a more specific CSS selector to avoid unintended styling of other elements. For example, `.user-info-container .user-info` or `.user-info-wrapper > .user-info`. 🎯